src/inmanta/app.py

```diff
@@ -363,6 +363,8 @@ def compile_project(options: argparse.Namespace) -> None:
         do_compile()
         LOGGER.debug("Compile time: %0.03f seconds", time.time() - t1)
 
+    project_instance.log_installed_modules()
```

Comment:
Wouldn't it make more sense to log this before starting the compile? Same for export.

Reply:
Agreed :+1: